release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -0,0 +1,852 @@
+instance of the custom metrics adapter will be overwritten in future releases,
+if {product-title} ships an out-of-the-box custom metrics adapter.
+
+[[ocp-4-1-cluster-monitoring-alerting-UI]]
+==== New alerting user interface
+
+An alerting UI is now natively integrated into the {product-title} web console.
+You can now view cluster-level alerts and alerting rules from a single place, as
+well as configure silences.
+
+[[ocp-4-1-cluster-monitoring-telemeter]]
+==== Telemeter
+
+Telemeter collects anonymized cluster-related metrics to proactively help
+customers with their {product-title} clusters. This helps:
+
+* Gather crucial health metrics of {product-title} installations.
+* Enable a viable feedback loop of {product-title} upgrades.
+* Enable metering.
+
+[[ocp-4-1-cluster-monitoring-autoscale]]
+==== Autoscale pods horizontally based on the resource metrics API
+
+By default, OpenShift Cluster Monitoring exposes CPU and Memory utilization
+through the Kubernetes resource metrics API. There is no need to install a
+separate metrics server anymore.
+
+[[ocp-4-1-developer-experience]]
+=== Developer experience
+
+[[ocp-4-1-code-ready-containers]]
+==== Code ready containers
+
+A local desktop instance of {product-title} 4.1 replaces the functions of `oc
+cluster` commands, Minishift, and CDK. {product-title} 4.1 focuses on ease of
+access and native experience, with a native installer on macOS and Microsoft
+Windows, native hypervisor support, and tray icon integration.
+
+[[ocp-4-1-registry]]
+=== Registry
+
+[[ocp-4-1-registry-managed-by-operator]]
+==== The registry is now managed by an Operator
+
+The registry is now managed by an Operator instead of `oc adm registry`.
+
+[[ocp-4-1-networking]]
+=== Networking
+
+[[ocp-4-1-cno]]
+==== Cluster Network Operator (CNO)
+
+The cluster network is now configured and managed by an Operator. The Operator
+upgrades and monitors the cluster network.
+
+[[ocp-4-1-openshift-sdn]]
+==== OpenShift SDN
+
+The default mode is now `NetworkPolicy`.
+
+[[ocp-4-1-multus]]
+==== Multus
+
+Multus is a meta plug-in for Kubernetes Container Network Interface (CNI), which
+enables a user to create multiple network interfaces per pod.
+
+[[ocp-4-1-web-console]]
+=== Web console
+
+[[ocp-4-1-developer-catalog]]
+==== Developer Catalog
+
+{product-title} 4.1 features a redesigned Developer Catalog that brings all of
+the new Operators and existing broker services together, with new ways to
+discover, sort ,and understand how to best use each type of offering. The
+Developer Catalog is the entry point for a developer to access all services
+available to them. It merges all capabilities from Operators, the Service
+Catalog, brokers, and Source-to-Image (S2I).
+
+[[ocp-4-1-operator-powered-UI-extensions]]
+==== Operator-powered user interface extensions
+
+Now with Operators, you can pick and choose what functionality you want to
+enable, allowing the user to customize the interface.
```

Comment:
Hi, @ahardin-rh. It's not clear to me what this means. Is this enabling features through operators or actual UI customizations? We plan to let admins create custom resources to customize the UI, but that won't be until at least 4.2.

Reply:
This was gathered from an early 4.x roadmap presentation, so I will drop this section.